gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -65,6 +66,13 @@ def active_servers_conninfo
       servers.map! { |info| db_yml_params.merge(info) }
     end
 
+    def raise_failover_event
+      require "awesome_spawn"
+      AwesomeSpawn.run("rake evm:raise_server_event",
+                       :chdir  => RAILS_ROOT,
```

Comment:
I feel like you should probably define it here because it's a bit odd that it defines a constant at the top level.